includes/class-scheduler.php

```diff
@@ -47,6 +46,8 @@ public static function init() {
 
 		// Event callbacks.
 		\add_action( 'activitypub_async_batch', array( self::class, 'async_batch' ), 10, 99 );
+		\add_action( 'activitypub_send_followers', array( self::class, 'async_batch' ), 10, 3 );
+		\add_action( 'activitypub_retry_followers', array( self::class, 'async_batch' ), 10, 3 );
```

Comment:
this sounds a lot like these were actions on the followers, not on Activites that will be sent to the followers 🤔

what about 

```suggestion
		\add_action( 'activitypub_send_activities', array( self::class, 'async_batch' ), 10, 3 );
		\add_action( 'activitypub_retry_activities', array( self::class, 'async_batch' ), 10, 3 );
```

Reply:
Oh, I like it, why did I not think of that. Let's go with the singular!